update-wpt.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
```

Comment:
I recommend putting this under `.github` folder and also adding a github workflow similar to: https://github.com/ada-url/ada/blob/main/.github/workflows/wpt-updater.yml

Reply:
I can make this change in another PR. I'd like to merge this one and do a release so @y3ti can get his ARM binaries (ref #30). Good with that?

Reply:
I'm good with all decisions you make in this project. I see you as the owner, and me as the "reviewer" making recommendations (not dictating them) 👍